libtransact/src/state/merkle/sql/backend/mod.rs

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Nitpick in commit message: These trait help => These traits help

Reply:
Fixed